Block/Configuration.php

```diff
@@ -186,7 +186,7 @@ public function getConfiguration()
             'maxValuesPerFacet' => (int) $config->getMaxValuesPerFacet(),
             'autofocus' => true,
             'request' => [
-                'query' => html_entity_decode($query),
+                'query' => htmlspecialchars(html_entity_decode($query)),
```

Comment:
shouldn't the function here be `htmlspecialchars_decode ` ?

Reply:
`htmlspecialchars` is correct here as we should be encoding it at this stage and decoding in js.

Reply:
yes, it makes sense.